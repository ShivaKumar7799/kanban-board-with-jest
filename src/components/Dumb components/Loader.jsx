import React from "react";

function Loader() {
  return (
    <div className="position-fixed w-100 z-index min-vh-100 d-flex justify-content-center align-items-center">
      <span className="loader"></span>
    </div>
  );
}

export default Loader;
