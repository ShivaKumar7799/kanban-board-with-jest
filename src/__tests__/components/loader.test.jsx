import React from 'react';
import { render } from '@testing-library/react';
import Loader from '../../components/Dumb components/Loader';

describe('Loader', () => {

  test("renders loader component correctly", () => {
    render(<Loader />)
  })

  test('snapshot of loader component', () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });

});