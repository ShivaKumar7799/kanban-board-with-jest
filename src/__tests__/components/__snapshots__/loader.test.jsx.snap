// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot of loader component 1`] = `
<div>
  <div
    class="position-fixed w-100 z-index min-vh-100 d-flex justify-content-center align-items-center"
  >
    <span
      class="loader"
    />
  </div>
</div>
`;
